crates/tick_probe/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use clap::Parser;
+use composer_api::{Client, Event};
+use eyre::Result;
+use std::{thread, time::Duration};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Frequency of the tick events to generate.
+    frequency: f64,
```

Comment:
Food for though: We're starting to express frequency at multiple [places](https://github.com/tonarino/acoustic_profiler/pull/26#discussion_r1177470107) in the probes' APIs. We may want unify how we do this.

There are two analog ways:
1. frequency (occurrences per second) as `f64`
2. period (duration between two occurrences) as `Duration`

I feel like my brain prefers to think in *frequencies* when frequency is high (1-5 Hz and above) and in periods when it's low: It's weird to to say "half a Hertz" but it's also not much intuitive to say "it happens every 4 millisecond" compared to "it happens at 250 Hz".

How do you feel about this?

Reply:
I agree with your general approach: it's always better to use whole numbers as they're easier to visualize, so let's go with whatever's easier in context.

In doubt, I find `Duration` easier to handle than frequency, both because hertz has no obvious strong type behind it (so it usually ends up stored in an easier to misplace u32), and because we generally have to convert frequencies into duration anyway, which is an unnecessary step if your constants are `Duration` to begin with.

Reply:
Agree with both of you. Here I opted for frequency because it is user-facing and I want to use this test probe to test that the actual _frequency_ of the sound generated (by overlaying samples) matches the passed one.